redash/query_runner/pg.py

```diff
@@ -72,9 +72,9 @@ def _wait(conn, timeout=None):
 
 def full_table_name(schema, name):
     if "." in name:
-        name = u'"{}"'.format(name)
+        name = '"{}"'.format(name)
```

Comment:
The actual changes in this file are in https://github.com/getredash/redash/pull/5052/commits/ea50fd9476489f11a57b8abc15d19f55d6db1802 the rest is just Black formatting